packaging/osx/buildpackage.sh

```diff
@@ -12,9 +14,12 @@ if [ -e "OpenRA.app" ]; then
     exit 2
 fi
 
+wget https://github.com/OpenRA/OpenRALauncherOSX/releases/download/${LAUNCHER_TAG}/launcher.zip || exit 3
```

Comment:
It probably does make sense though, since this script probably will work if run directly.